sqe-database-access/CatalogueRepository.cs

```diff
@@ -87,11 +87,10 @@ public async Task<IEnumerable<CatalogueMatch>> GetImagedObjectMatchesForTextFrag
             string canonicalEditionName, string canonicalEditionVolume, string canonicalEditionLoc1,
             string canonicalEditionLoc2, byte canonicalEditionSide, string comment)
         {
-            using (var transactionScope = new TransactionScope())
+            using (var transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
             using (var connection = OpenConnection())
             {
-                //var existingMatches = await GetTextFragmentMatchesForImagedObjectAsync(imagedObjectId);
-                var existingEditionCats = await connection.QueryAsync<EditionCatalogueEntry>(
+                var existingEditionCats = (await connection.QueryAsync<EditionCatalogueEntry>(
```

Comment:
Wouldn't it be simpler to change QueryAsync to call ToList() before returning the value?

Reply:
I guess so, I just didn't want to create too many abstractions.  I guess I made a bit of a mistake here and should have called Dapper's AsList(), though it probably doesn't matter.